eng/pipelines/extra-platforms/runtime-extra-platforms-other.yml

```diff
@@ -238,6 +238,38 @@ jobs:
       extraVariablesTemplates:
         - template: /eng/pipelines/common/templates/runtimes/test-variables.yml
 
+#
+# Mono CoreCLR runtime test executions using live libraries and mini Full AOT
```

Comment:
This should be enabled in `runtime-llvm.yml` as well. Ideally, that file and this file should share the logic in someway in terms of this specific CI lane logic, which I don't know how.

Reply:
The changes in this PR are focused on mini backend only. Do you know what is the purpose of the `runtime-llvm.yml`?

Reply:
oh I see. I thought this PR was for LLVM full aot. Please ignore my previous comment.